universal-application-tool-0.0.1/app/services/question/AddressQuestionDefinition.java

```diff
@@ -0,0 +1,36 @@
+package services.question;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
+
+public class AddressQuestionDefinition extends QuestionDefinition {
```

Comment:
Issue (blocking): this class as well as its superclass represent a question type, rather than a specific question definition.

Suggestion: rename `*QuestionDefinition` to `*QuestionTypeDefinition`

Reply:
Errr wait no they don't, sorry.